explorer-api/src/state.rs

```diff
@@ -1,20 +1,40 @@
-use chrono::{DateTime, Utc};
-use serde::{Deserialize, Serialize};
 use std::fs::File;
 use std::path::Path;
 
+use chrono::{DateTime, Utc};
```

Comment:
Any reason for chrono here?

Reply:
It was so the response was formatted as an iso timestamp - I could change it to use `humantime-serde` if Chrono disgusts you?

Reply:
Oh, chrono is absolutely fine by me - I just couldn't see any of its usages in the code so I assumed it was a dead dependency : )